string ades_name
---
MotionSequence[] motion_sequences
